helm/botkube/values.yaml

```
@@ -116,25 +118,25 @@ sources:
     # -- Describes Kubernetes source configuration.
     # @default -- See the `values.yaml` file for full object.
     botkube/kubernetes:
-      context: &defaultPluginContext
+      context: &default-plugin-context
         defaultNamespace: "default"
         # -- RBAC configuration for this plugin.
         rbac:
           # -- Static impersonation for a given username and groups.
-          group:
+          user:
             type: Static
             # -- Prefix that will be applied to .static.value[*].
             prefix: ""
             static:
-              # -- Name of group.rbac.authorization.k8s.io the plugin will be bound to.
-              values: [*static-group-name] # "botkube-plugins-read-only" is the default
-          user:
+              # -- Name of user.rbac.authorization.k8s.io the plugin will be bound to.
+              value: *static-group-name
+          group:
             type: Static
             # -- Prefix that will be applied to .static.value[*].
             prefix: ""
             static:
-              # -- Name of user.rbac.authorization.k8s.io the plugin will be bound to.
-              value: *static-group-name
+              # -- Name of group.rbac.authorization.k8s.io the plugin will be bound to.
+              values: [*static-group-name] # "botkube-plugins-read-only" is the default
```

```suggestion
              values: [*static-group-name] # "botkube-plugins-read-only" is the default
```
Is `static-group-name` anchor used anywhere? If not, please remove it.